5_databases/week1/homework.sql

```diff
@@ -0,0 +1,55 @@
+-- Find out how many tasks are in the task table
+SELECT count(id) 
+FROM task;
+
+-- Find out how many tasks in the task table do not have a valid due date
+SELECT count(id)
+FROM task
+WHERE due_date IS NULL;
+
+-- Find all the tasks that are marked as done
+SELECT *
+FROM task
+WHERE status_id = 3;
+
+-- Find all the tasks that are not marked as done
+SELECT *
+FROM task
+WHERE status_id != 3;
+
+--Get all the tasks, sorted with the most recently created first
+SELECT *
+FROM task
+ORDER BY created DESC;
+
+-- Get the single most recently created task
+SELECT *
+FROM task
+ORDER BY created DESC
+LIMIT 1;
+
+--Get the title and due date of all tasks where the title or description contains database
+SELECT title , due_date
+FROM task
+WHERE title LIKE '%database%' OR description LIKE '%database%';
+
+-- Get the title and status (as text) of all tasks
+SELECT t.title , s.name
```

Comment:
I've never seen such syntax. As I see from previous queries your tables' names are task and status but here you used t and s. I don't think SQL will accept that 🤔

Reply:
I learned this syntax from youtube and it worked for me. but if you think it is not good I will change that.

Reply:
No, I think you can leave it. It's cool that you were trying to learn more about SQL. I just wanted to point out that in real situation this probably won't work because it's hard to understand. sometimes short syntax is a bad solution. You can talk to Lawan about this approach and let him give his opinion about such an approach.  But I wouldn’t use this in my code just because it's very easy to mess up something 😊